test/test_suite.go

```diff
@@ -133,8 +136,7 @@ func ConcurrentReadWriteTest(t *testing.T, d da.DA) {
 	go func() {
 		defer wg.Done()
 		for i := uint64(1); i <= 100; i++ {
-			_, err := d.GetIDs(ctx, i, []byte{})
-			assert.NoError(t, err)
+			_, _ = d.GetIDs(ctx, i, []byte{})
```

Comment:
I would really like to see that only the nitpickly-introduced error is allowed here.

Reply:
fair